assets/src/edit-story/elements/image/index.js

```diff
@@ -29,7 +29,7 @@ export { default as LayerIcon } from './icon';
 export const defaultAttributes = {
   scale: 100,
   focalX: 50,
-  focalY: 50,
+  focalY: 50
```

Comment:
Wasn't there a linter that always inserts the trailing commas?

Reply:
There should be 🤔

Reply:
My bad, prettier was disabled